crowbar_framework/spec/controllers/crowbar_controller_spec.rb

```diff
@@ -76,20 +76,20 @@
     end
 
     it "transitions the node into desired state" do
-      RoleObject.stubs(:find_roles_by_search).returns([])
+      allow_any_instance_of(RoleObject).to receive(:find_roles_by_search).and_return([])
       post :transition, barclamp: "crowbar", id: "default", state: "discovering", name: "testing"
       expect(response).to be_success
     end
 
     it "returns plain text message if transitioning fails" do
-      CrowbarService.any_instance.stubs(:transition).returns([500, "error"])
+      allow_any_instance_of(CrowbarService).to receive(:transition).and_return([500, "error"])
       post :transition, barclamp: "crowbar", id: "default", state: "discovering", name: "testing"
       expect(response).to be_server_error
       expect(response.body).to be == "error"
     end
 
     it "returns node as a hash on success when passed a name" do
-      CrowbarService.any_instance.stubs(:transition).returns([200, { name: "testing" }])
+      allow_any_instance_of(CrowbarService).to receive(:transition).and_return([200, { name: "testing" }])
```

Comment:
Metrics/LineLength: Line is too long. [106/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)